src/histolab/tiler.py

```diff
@@ -48,7 +43,6 @@ class Tiler(Protocol):
     level: int
     tile_size: int
 
-    @lru_cache(maxsize=100)
```

Comment:
why did you remove it?

Reply:
i misunderstood something ... sorry ... restored right now